lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -380,9 +380,17 @@ class ArynPartitioner(Partitioner):
              Ignored when local mode is false.
         extract_images: If true, crops each region identified as an image and attaches it to the associated
              ImageElement. This can later be fed into the SummarizeImages transform.
+        device: Device on which to run the partitioning model locally. One of 'cpu', 'cuda', and 'mps'. If
+             not set, Sycamore will choose based on what's available. If running remotely, this doesn't
+             matter.
+        batch_size: How many pages to partition at once. Default is 1.
+        batch_at_a_time: When running remotely, send only one batch at a time. Default is False
```

Comment:
batch_at_a_time: When running locally, True means inference is run on the pages in batches - not all pages are loaded into memory at the same time.